infra/scripts/test-end-to-end-batch-dataflow.sh

```diff
@@ -0,0 +1,272 @@
+============================================================
+"
+export TEMP_BUCKET=$TEMP_BUCKET
+export DATASET_NAME=$DATASET_NAME
+export GCLOUD_PROJECT=$GCLOUD_PROJECT
+export GCLOUD_NETWORK=$GCLOUD_NETWORK
+export GCLOUD_SUBNET=$GCLOUD_SUBNET
+export GCLOUD_REGION=$GCLOUD_REGION
+
+echo "
+============================================================
+Helm install Feast and its dependencies
+============================================================
+"
+cd $ORIGINAL_DIR/infra/scripts/test-templates
+envsubst $'$TEMP_BUCKET $DATASET_NAME $GCLOUD_PROJECT $GCLOUD_NETWORK \
+  $GCLOUD_SUBNET $GCLOUD_REGION $feast_kafka_1_ip
+  $feast_kafka_2_ip $feast_kafka_3_ip $feast_redis_ip $feast_statsd_ip' < values-end-to-end-batch-dataflow.yaml > $ORIGINAL_DIR/infra/charts/feast/values-end-to-end-batch-dataflow-updated.yaml
+
+cd $ORIGINAL_DIR/infra/charts
+helm template feast
+
+cd $ORIGINAL_DIR/infra/charts/feast
+helm install --wait --timeout 600s --values="values-end-to-end-batch-dataflow-updated.yaml" ${HELM_RELEASE_NAME} .
+
+echo "
+============================================================
+Installing Python 3.7 with Miniconda and Feast SDK
+============================================================
+"
+# Install Python 3.7 with Miniconda
+wget -q https://repo.continuum.io/miniconda/Miniconda3-4.7.12-Linux-x86_64.sh \
+   -O /tmp/miniconda.sh
+bash /tmp/miniconda.sh -b -p /root/miniconda -f
+/root/miniconda/bin/conda init
+source ~/.bashrc
+
+# Install Feast Python SDK and test requirements
+cd $ORIGINAL_DIR
+make compile-protos-python
+pip install -qe sdk/python
+pip install -qr tests/e2e/requirements.txt
+
+echo "
+============================================================
+Running end-to-end tests with pytest at 'tests/e2e'
+============================================================
+"
+# Default artifact location setting in Prow jobs
+LOGS_ARTIFACT_PATH=/logs/artifacts
+
+cd $ORIGINAL_DIR/tests/e2e
+
+core_ip=$(kubectl get -o jsonpath="{.spec.clusterIP}" service ${HELM_RELEASE_NAME}-feast-core)
+serving_ip=$(kubectl get -o jsonpath="{.spec.clusterIP}" service ${HELM_RELEASE_NAME}-feast-batch-serving)
+
+set +e
+pytest bq-batch-retrieval.py -m dataflow_runner --core_url "$core_ip:6565" --serving_url "$serving_ip:6566" --gcs_path "gs://${TEMP_BUCKET}/" --junitxml=${LOGS_ARTIFACT_PATH}/python-sdk-test-report.xml
+TEST_EXIT_CODE=$?
+
+if [[ ${TEST_EXIT_CODE} != 0 ]]; then
+  echo "[DEBUG] Printing logs"
+  ls -ltrh /var/log/feast*
+  cat /var/log/feast-serving-warehouse.log /var/log/feast-core.log
+
+  echo "[DEBUG] Printing Python packages list"
+  pip list
+fi
+
+cd ${ORIGINAL_DIR}
+exit ${TEST_EXIT_CODE}
+
+echo "
+============================================================
+Cleaning up
+============================================================
+"
+cd $ORIGINAL_DIR/tests/e2e
+
+# Remove BQ Dataset
+bq rm -r -f ${GCLOUD_PROJECT}:${DATASET_NAME}
+
+# Clean up GKE cluster
+kubectl delete services --all
```

Comment:
Which namespace is this limited to? How do you know you are deleting the right services?